client/src/pages/Profile.js

```diff
@@ -83,6 +85,19 @@ const Profile = () => {
     });
   };
 
+  const createUpdatedUser = () => {
+    const updatedUser = {};
+    userPermissions.edit.forEach((field) => {
+      updatedUser[field] = user[field];
+    });
+
+    return updatedUser;
+  };
+
+  const submitChanges = () => {
+    updateMember(createUpdatedUser(), user._id);
+  };
+
   return (
```

Comment:
Just wondering, would it be possible for you to add a "status" alert here that appears after you submit call updateMember? Since the backend returns "message" in the response, you can just put the contents of "message" in the alert. It can be a red alert when success is false, and a green alert when success is true. Right now there's no confirmation that the member got updated which is a bit confusing.